### DISCUSSION TITLE
Aiohttp support?

### DISCUSSION THREAD

**Discussion by snowy-jaguar** [encode/starlette#1545] [Q&A] [answered]
So I see in the dependencies Starlet uses requests for its test client, is there a aiohttp equivalent?

**Comment by Kludex** [comment, marked as answer]
I don't think so. 🤔

**Comment by taybin** [comment]
There is a ticket https://github.com/encode/starlette/issues/652 to use [httpx](https://www.python-httpx.org/) as an async http client in tests.
